youtube_dl/extractor/kommunetv.py

```diff
@@ -6,7 +6,8 @@
 
 
 class KommunetvIE(InfoExtractor):
-    _VALID_URL = r'https://(\w+).kommunetv.no/archive/(?P<id>\w+)'
+    _VALID_URL = r'https?://\w+\.kommun(?:etv\.no|\.tv)/(?:archive|live)/(?P<id>\w+)'
+
     _TEST = {
```

Comment:
Make this 
```py
    _TESTS = [{
...
    }]
```
and add tests for the various URL types.

Reply:
Yes, I have considered doing something like that, but I'm unsure how to obtain the MD5 values.

Alternatively, I could create a new function specifically for live streams.

Reply:
The MD5 values may not be critical, especially if the site is sending adaptive formats where the value could vary. Anyhow, just draft a test case with `url`, `id`, `ext`; the test harness will tell you what other values must be included. For optional items like `md5`, include a dummy value and the test harness will show the correct value.